pkgs/development/tools/scaff/default.nix

```diff
@@ -1,26 +1,27 @@
 { lib, rustPlatform, fetchFromGitLab, pkg-config, openssl }:
 
-rustPlatform.buildRustPackage rec {
-  pname = "scaff";
-  version = "0.1.2";
+builtins.trace "scaff is deprecated - replaced by https://gitlab.com/jD91mZM2/inc (not in nixpkgs yet)" (
```

Comment:
Eval warnings of this kind are not currently accepted in nixpkgs

Reply:
What's the best way to deprecate a package?

Reply:
Remove it and add an alias to aliases.nix.